src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -408,46 +353,8 @@ namespace OpenLoco
             addr<0x005233B2, int32_t>() += addr<0x01140840, int32_t>();
             addr<0x0114084C, int32_t>() = 0;
             addr<0x01140840, int32_t>() = 0;
-            if (Config::get().old.var_72 == 0)
-            {
-                Config::get().old.var_72 = 16;
-                const auto cursor = Ui::getCursorPosScaled();
-                addr<0x00F2538C, Ui::Point32>() = cursor;
-                drawingCtx.clear(0);
-                addr<0x00F2539C, int32_t>() = 0;
-            }
-            else
-            {
-                if (Config::get().old.var_72 >= 16)
-                {
-                    Config::get().old.var_72++;
-                    if (Config::get().old.var_72 >= 48)
-                    {
-                        if (sub_4034FC(addr<0x00F25394, int32_t>(), addr<0x00F25398, int32_t>()))
-                        {
-                            uintptr_t esi = addr<0x00F25390, int32_t>() + 4;
-                            esi *= addr<0x00F25398, int32_t>();
-                            esi += addr<0x00F2538C, int32_t>();
-                            esi += 2;
-                            esi += addr<0x00F25394, int32_t>();
-                            addr<0x00F2539C, int32_t>() |= *((int32_t*)esi);
-                            call(0x00403575); // ddrwaUnlockPSurface
-                        }
-                    }
-                    Ui::setCursorPosScaled(addr<0x00F2538C, int32_t>(), addr<0x00F25390, int32_t>());
-                    Gfx::invalidateScreen();
-                    if (Config::get().old.var_72 != 96)
-                    {
-                        return;
-                    }
-                    Config::get().old.var_72 = 1;
-                    if (addr<0x00F2539C, int32_t>() != 0)
-                    {
-                        Config::get().old.var_72 = 2;
-                    }
-                    Config::write();
-                }
 
+            {
```

Comment:
To prevent cluttering the diff, I just this into a simple scoping block, for now.

Reply:
call(0x00452D1A); // nop redrawPeepAndRain

could also be removed. It handles the redrawing peeps when picked up (which isn't an issue for loco) and handles drawing rain (again not an issue for loco)